healthz/healthz.proto

```diff
@@ -41,6 +42,20 @@ service Healthz {
   // Get will get health status for a gNMI path.  If no status is available for
   // the requested path an error will be returned.
   rpc Get(GetRequest) returns (GetResponse) {}
+
+  // List returns all events for the provided component.
```

Comment:
Should this be `provided path`, to be consistent with the line above?

Reply:
done